components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -94,15 +94,6 @@ func installWorkspaceRoutes(r *mux.Router, config *RouteHandlerConfig, ip Worksp
 	})
 	routes.HandleSupervisorFrontendRoute(faviconRouter.NewRoute())
 
-	// Theia has a bunch of special routes it probably requires.
```

Comment:
Are we removing these lines in scope of fixing vscode or just because theia is deprecated now?

Reply:
because is deprecated